README.md

```diff
@@ -1,2 +1,45 @@
 # MobilePortfolio
-Personal portfolio project
+Personal portfolio project for Microverse course (Module 1)
+
+The aim of this project is to learn to understand how to parse a Figma design, use flexbox to place elements in the page, build a personal portfolio site.
+
+## Built With
+
+- HTML
+- css
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Author: Maria Amalia Gomez Moro
+1. GitHub: @AmaliaGM
+2. Twitter: @AmaliaGomezMoro
+3. Linkedin: @MariaAmaliaGomezMoro
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+-    To my son for putting up with me through Microverse Course.
+-    To my coding partner.
+-    To my mentor.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The MIT License does not exist in the repo please consider adding it so the link is working